pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx

```diff
@@ -448,13 +449,13 @@ function SpaceInvites({ spaceId }: { spaceId: SpaceId; }): ReactElement {
 						}
 					</tbody>
 				</table>
-				{ showCreation && <SpaceInviteCreation closeDialog={ () => setShowCreation(false) } addInvite={ addInvite } /> }
+				{ showCreation && <SpaceInviteCreation closeDialog={ () => setShowCreation(false) } addInvite={ addInvite } isPlayerAdmin={ isPlayerAdmin } /> }
 			</Column>
 		</FieldsetToggle>
 	);
 }
 
-function SpaceInviteCreation({ closeDialog, addInvite }: { closeDialog: () => void; addInvite: (invite: SpaceInvite) => void; }): ReactElement {
+function SpaceInviteCreation({ closeDialog, addInvite, isPlayerAdmin }: { closeDialog: () => void; addInvite: (invite: SpaceInvite) => void; isPlayerAdmin: boolean; }): ReactElement {
 	const directoryConnector = useDirectoryConnector();
 	const [allowAccount, setAllowAccount] = useState(false);
```

Comment:
Nitpick: This should default to `!isPlayerAdmin`

Reply:
why?
the function is only ever called locally (not exported) where this is provided as a variable and not a literal
if we ever need `<SpaceInviteCreation isPlayerAdmin={ false } />` directly only then do I see a reason for this
the function above this `SpaceInvites` is the same

Reply:
Because it is not possible for non-admins to create an invite that doesn't make use of account filtering - so if player is not admin, they always need to enable that filter to successfully create an invite that way.
Another alternative I'm thinking about is just removing this menu altogether for non-admins and making this menu only create space-bound invites.

But I don't feel strongly about this (I wrote it is a nitpick in the first place) - feel free to close this thread.

Reply:
ahh I see, wasn't quite sure about the reasoning, well I just wanted to provide an alternative to `/invite` even if it is a bit raugh~~

in any case I think it is good for now and we can refine it later